index.html

```diff
@@ -9,41 +9,41 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Open+Sans:300">
 </head>
     <body>
-        <section class = "exp_the_rush">
-            <div class = "container">
-                <h1>Experience the rush today</h1>
-                <p class="text">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
+        <section class = "exp-the-rush-section">
+            <div class = "exp-the-rush-section__main-container main-container">
+                <h1 class= "exp-the-rush-section__title">Experience the rush today</h1>
+                <p class="exp-the-rush-section__desc">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
                     magna aliqua consectetur adipisicing elit.</p>
-                <div class = "button_container">
-                    <button class = "main_button">Get started</button>
+                <div class = "exp-the-rush-section__button-container">
+                    <button class = "exp-the-rush-section__main_button main-button">Get started</button>
                 </div>
             </div>
         </section>
         <section class = "features-section">
-            <div class = "container">
-                <dl class="feature">
-                    <div class="feature__icon_container">
-                        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACQAAAAnCAMAAABQShoEAAAAe1BMVEUAAACAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHqAQHrRmSaKAAAAKHRSTlMAB9GG+qRADeHcmEUvE/WNSqp4OScf1762klrv5+OxbFQZfmZNYzTKuEpQVwAAAZRJREFUOMuFlNlygzAMRQ0xYNawBAgBQtIsPf//hSWFtoYA1QuyfEa6up5BvMXlfBD/hsnuN793i8g1UyRPZzxJrDlgXKyGIYoqe1UsZpSRNuhhXvpaMqXiFjB3ce6IMDh5DVDmwkKn9kAS6KNNqEtQ4I2lMxTZkGf7cEhuEZD0V9Fw7pnkZ6NPPsbsUEDXa73/zPKWfApNpD3mPiTLZl4VbTgIcnGdZUhkcpzRgS1WoF6J/H5KxVGsQmGNNyiy16G+VW0I4eGKDegAvQUu+w1ouHfgpNc+JO1dL1iUIoBAK+0ApC4ypX3pvv5VnIib703WfRCJGJyJzECkk1VOIOx5p/RiTTrdiJY1ZfomtG/bpQ2mP99OFFRiPUYfK9QWE0AsXsr9DaiiNr77letMGFENOxKvQh4yf32NAhWuMDZUYyY5GotM3qCcX7+wlpirq/tqwfF94lNNjDYsUDP1RiqRj0lpBxxt7aUfLtTxfI8CUN7pmYe53SURcM7FPIxOoUe58g6+5w6A/Nxt/V2dwI/tw8y1LyMdLfl80m0NAAAAAElFTkSuQmCC" alt="timer icon"/>
+            <div class = "features-section__main-container main-container">
+                <dl class="features-section__feature feature">

```

Comment:
I can't find `features-section__feature` in css. Does it make any sense to have it here?

Reply:
Yeah, as for me, assigning classes for all components and nestings gives better structure understanding.